client/js/libs/handlebars/parse.js

```diff
@@ -10,6 +10,7 @@ const colorClass = require("./colorClass");
 const emojiMap = require("../fullnamemap.json");
 const LinkPreviewToggle = require("../../../components/LinkPreviewToggle.vue").default;
 const emojiModifiersRegex = /[\u{1f3fb}-\u{1f3ff}]/gu;
+const emojiVariationsRegex = /[\u{fe0e}-\u{fe0f}]/gu;
```

Comment:
I think it's better to combine it with previous regex for perf reasons.

Could you also add some tests for this specifically? So we know emojis with fe0e and without match correctly.

Reply:
Okay sounds good. And I'll try adding some tests, but I don't use Node and don't have it set up, although I think it should be easy to copy and modify one of the existing tests though.